src/FeedIo/Feed.php

```diff
@@ -146,6 +149,18 @@ public function add(ItemInterface $item) : FeedInterface
         return $this;
     }
 
+    public function addNS($ns, $dtd) : FeedInterface
+    {
+        $this->ns->append([$ns, $dtd]);
```

Comment:
I think each namespace must be unique, so maybe another approach would be to set the namespace as the offset name, with the `offsetSet()` method